scr/src/main/java/org/apache/felix/scr/impl/metadata/ReferenceMetadata.java

```diff
@@ -101,7 +101,11 @@ public enum ReferenceScope {bundle, prototype, prototype_required}
 
     public static final String CONDITION_TRUE_FILTER = "(osgi.condition.id=true)";
 
-    // TODO this constant will be defined in the R8 Declarative Services spec
+    /**
+     * The reference name for the implicit satisfying condition as defined in the
+     * OSGi R8 Declarative Services specification (see https://github.com/osgi/osgi/pull/875).
+     * This reference is automatically added to components when a true condition service is available.
+     */
```

Comment:
The point of this comment wasn't to add javadoc later to this internal class.  It was to remind us to use the actual spec'ed constant from the released APIs
 https://github.com/osgi/osgi/blob/eb08fcac898f50aec1a1d2847cae6ffccd5a7385/org.osgi.service.component/src/org/osgi/service/component/ComponentConstants.java#L161

Reply:
I created
- https://github.com/apache/felix-dev/pull/454